source/elements/oneDAL/source/data_management/tables.rst

```diff
@@ -1,3 +1,474 @@
+.. highlight:: cpp
+
 ======
 Tables
 ======
+
+Table is a generic |dal_short_name| concept over numerical data. It provides uniformed way
+to pass the data to the library as :ref:`inputs <Input>` or :ref:`parameters <Descriptor>` or get them as :ref:`results <Result>`.
+
+Table object is a container of two entities: data and metadata.
+
+- Data is organized in a shape of :math:`(N \times p)`,
+  where :math:`N` is a number of :term:`observations <Observation>` in a table and :math:`p`
+  is a number of :term:`features <Feature>`.
+
+- Metadata defines the detailed structure of the data and,
+therefore, helps |dal_short_name| to access the data efficiently
+(see :ref:`Metadata API` section for details).
```

Comment:
add spaces in the beginning of each line as it's done above, otherwise sphinx throws an error